### DISCUSSION TITLE
Pocketbase server asking if I'm gay

### DISCUSSION THREAD

**Discussion by suelitonx** [pocketbase/pocketbase#4601] [Q&A]
Hello everything is fine? I deployed Pocketbase through [Digital Ocean](https://marketplace.digitalocean.com/apps/pocketbase). There is a problem where I cannot use pocketbase in any way. It's as if the server went down. Below is a printout with the logs. Has anyone gone through this who can help me? Note: I deployed to 2 different accounts and the problem happened on both.

![Screenshot_1](https://github.com/pocketbase/pocketbase/assets/145073364/ee3b746a-2dca-40cf-a519-43d327620416)
![Screenshot_4](https://github.com/pocketbase/pocketbase/assets/145073364/39efce65-9bb5-4918-90ba-370161ccdd64)

Edit: I'm using Pocketbase on Flutter, when trying to log in, it always gives an error. And these strange logs happen.
![Screenshot_2](https://github.com/pocketbase/pocketbase/assets/145073364/8030ff3f-007f-4964-85c8-58bcf5cca97b)

**Comment by ganigeorgiev** [comment]
I don't understand what do you mean by _"I cannot use pocketbase in any way. It's as if the server went down"_.

The above are just requests by bot/scanners. They shouldn't affect the normal operation of your server and I don't think your Flutter error has anything to do with them (_unless someone is fludding your server with so much requests that it cannot respond, but if that was the case then the Admin UI wouldn't be accessible which from the screenshot seems to work fine; additionally DO and other similar providers usually should have a [builtin DDoS protection](https://www.digitalocean.com/products/ddos-protection)_).

You can safely ignore them or block the IPs via iptables or similar.
If you have the time, you can also try to find the owner of the ips via various ip lookup tools and report them to the abuse address that usually comes with the info.

But again, your Flutter error is most likely not related to the above logs.
